src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGSequenceManager.cs

```diff
@@ -35,9 +35,6 @@ public class RGSequenceManager : MonoBehaviour
 
     private ReplayToolbarManager _replayToolbarManager;
 
-    private bool _loadingSequences = false;
```

Comment:
Removing dead variables from this PR https://github.com/Regression-Games/RGUnityBots/pull/301